services/io_services/io_read_pokemon.go

```diff
@@ -0,0 +1,56 @@
+package io_services
+
+import (
+	"encoding/csv"
+	"fmt"
+	"jmc/bootcamp/models"
+	"os"
+)
+
+type CsvService struct {
+  PathFile string
+}
+
+func (srv CsvService)ReadFromService() (pokemonList []models.Pokemon, err error) {
```

Comment:
You can fill and return a map saving the `Pokemons` by id, this will help to filter and display them.
```suggestion
func (srv CsvService)ReadFromService() (pokemonList map[int]models.Pokemon, err error) {
```